src/layer/tile/GridLayer.js

```diff
@@ -272,6 +272,17 @@ export const GridLayer = Layer.extend({
 		return document.createElement('div');
 	},
 
+	// @method refreshTiles: void
+	// Triggers creating of new tile HTML element for each existing tile
```

Comment:
Compare with the `redraw` method. Give users some explanation (in the docstrings) why they should use one or the other.

Reply:
explanation extended